http-client-openssl/Network/HTTP/Client/OpenSSL.hs

```diff
@@ -38,6 +38,7 @@ opensslManagerSettings mkContext = defaultManagerSettings
                 $ \sock -> do
                     N.connect sock address
                     ssl <- SSL.connection ctx sock
+                    SSL.setTlsextHostName ssl host'
```

Comment:
Can you add a minor version bump and a changelog entry explaining the change?

Reply:
Certainly!